public/js/actions/types.js

```diff
@@ -1,13 +1,49 @@
 // @flow
 
+/**
+ * Flow types
+ * @module actions/types
+ */
+
+/**
+ * Tri-state status for async operations
+ *
+ * Available options are:
+ * `"start"` or `"done"` or `"error"`
+ *
+ * @memberof actions/types
+ * @static
+ * @enum
+ */
 export type AsyncStatus = "start" | "done" | "error";
 
+/**
+ * Source File Location
+ *
+ * @memberof actions/types
+ * @static
+ * @property {string}  sourceId - Source ID for the file
+ * @property {number}  line     - Line number, (zero based?) within the file
+ * @property {number}  column   - Column number within the line (optional)
+ */
 export type Location = {
   sourceId: string,
   line: number,
   column?: number
 };
 
+/**
+ * Breakpoint
+ *
+ * @memberof actions/types
+ * @static
+ * @property {string}  id        - Source ID for the file
+ * @property {Location} location - Line number, (zero based?) within the file
+ * @property {boolean}  loading  - Loading status
+ * @property {boolean}  disabled - Disabled by the user
+ * @property {string}  text      - Text representation of relevant line
+ * @property {string}  condition - Conditional breakpoint value (optional)
```

Comment:
Yeah, I really don't like these doc strings now that we are using Flow. I was excited to get away from them in the devtools codebase because we have real Flow types. I haven't look at this system at all but I thought it supported flow, meaning that it read flow types? It makes me sad to see our nice and clean types have these comments :(